karma/setup/pubsub.ts

```diff
@@ -0,0 +1,103 @@
+import * as PubSub from "pubsub-js";
```

Comment:
>PubSub [](start = 12, length = 6)

All this appears to be boiler plate code. Can we not get it from npm package instead of manually copy pasting?

Reply:
I'm not sure what you mean, can you clarify ?

---
In reply to: [253637913](https://github.com/Microsoft/clarity-js/pull/132#discussion_r253637913) [](ancestors = 253637913)